core/src/lib/base64.h

```diff
@@ -36,3 +36,4 @@ int ToBase64(int64_t value, char* where);
 int FromBase64(int64_t* value, char* where);
 int BinToBase64(char* buf, int buflen, char* bin, int binlen, bool compatible);
 int Base64ToBin(char* dest, int destlen, char* src, int srclen);
+int Base64LengthUnpadded(const int source_length);
```

Comment:
The function is passed a copy of the int anyway, so making it `const` only affects the implementation, so we usually don't qualify by-value parameters with `const`.
```suggestion
int Base64LengthUnpadded(int source_length);
```